wolfcrypt/src/memory.c

```diff
@@ -1133,3 +1133,141 @@ void __attribute__((no_instrument_function))
 }
 #endif
 
+#ifdef WOLFSSL_LINUXKM_SIMD_X86_IRQ_ALLOWED
+union fpregs_state **wolfcrypt_irq_fpu_states = NULL;
+#endif
+
+#if defined(WOLFSSL_LINUXKM_SIMD_X86) && defined(WOLFSSL_LINUXKM_SIMD_X86_IRQ_ALLOWED)
+
+    static __must_check inline int am_in_hard_interrupt_handler(void) {
+        return (preempt_count() & (NMI_MASK | HARDIRQ_MASK)) != 0;
+    }
+
+    __must_check int allocate_wolfcrypt_irq_fpu_states(void) {
```

Comment:
Would prefer our functions conform with existing coding patterns with leading brace on newline.

Reply:
Oh darn, I missed this one before sending commits off to Jenkins.  I'll fix in the next round of changes.

Reply:
done now.